operators/multiclusterobservability/manifests/base/grafana/dash-acm-openshift-virtualization-single-cluster-view.yaml

```
@@ -0,0 +1,2737 @@
+apiVersion: v1
+data:
+  acm-openshift-virtualization-single-cluster-view.json: |-
+    {
+      "annotations": {
+        "list": [
+          {
+            "builtIn": 1,
+            "datasource": "-- Grafana --",
+            "enable": true,
+            "hide": true,
+            "iconColor": "rgba(0, 211, 255, 1)",
+            "name": "Annotations & Alerts",
+            "target": {
+              "limit": 100,
+              "matchAny": false,
+              "tags": [],
+              "type": "dashboard"
+            },
+            "type": "dashboard"
+          }
+        ]
+      },
+      "description": "This dashboard provides comprehensive insights into the cluster with OpenShift Virtualization installed. It offers detailed information about the virtual machines (VMs), associated resources, and any active alerts, enabling efficient monitoring and management of virtualization within the cluster.",
```

If we plan to use VMs. The first instance in previous files must be written out completely followed by the abbreviation. After that VM can be used. My previous suggestions were to just increase consistency
```suggestion
      "description": "This dashboard provides comprehensive insights of the cluster with OpenShift Virtualization installed. It offers detailed information about the virtual machines (VMs), associated resources, and any active alerts, enabling efficient monitoring and management of virtualization within the cluster.",
```